docs/guides/deep_linking/smart-banner-web-tool/dl_smart_banner_v2.md

```diff
@@ -18,8 +18,12 @@ The Smart Banner SDK authenticates using the unique **Web key**, which you can g
 
 ### Smart banners only
 
-1. Replace the _YOUR_WEB_KEY_ placeholder in the script with your **Web key**. The web key is created when you create a new [Website workplace](https://support.appsflyer.com/hc/en-us/articles/360000764837#1-website-setup).
-2. Paste the following code snippet in the `head` tag on your website. Make sure to paste it near the top of the `head` tag.
+You can either copy the smart banner script from AppsFlyer or from below.
+
+1. Copy the script code snippet in **one** of the following ways:
+
+   - Copy the script from the [smart banner’s Website workplace](https://support.appsflyer.com/hc/en-us/articles/360000764837-Smart-Banners-mobile-web-to-app-#1-website-workplace-setup) in AppsFlyer. Your web key is already included in the script.
```

Comment:
The capitalisation is weird.
Should be "Smart Banner's website workplace". Although I'm not sure we actually called that "website workplace"